generator.js

```diff
@@ -11,23 +11,24 @@ const moodTwo = 'happy'
 const moodThree = 'sleepy'
 const moodFour = 'sad'
 
-const bodyArr = [bodyOne,bodyTwo,bodyThree];
-const colorArr = [colorOne,colorTwo,colorThree,colorFour,colorFive];
+const bodyArr = [bodyOne, bodyTwo, bodyThree];
+const colorArr = [colorOne, colorTwo, colorThree, colorFour, colorFive];
 const moodArr = [moodOne, moodTwo, moodThree, moodFour];
 
-function generator (arr1,arr2,arr3) {
-    
-    let ran1 = Math.floor((Math.random() * arr1.length));
-    let ran2 = Math.floor((Math.random() * arr2.length));
-    let ran3 = Math.floor((Math.random() * arr3.length));
+function generator (arr1, arr2, arr3) {
+    let newBody = arrayRandomizer(arr1)
+    let newColor = arrayRandomizer(arr2)
+    let newMood = arrayRandomizer(arr3)
 
-    let newBody = bodyArr[ran1]
-    let newColor = colorArr[ran2]
-    let newMood = moodArr[ran3]
+    console.log(`Your monster is ${newBody}, ${newColor}, and ${newMood}!`);
+}
 
-    console.log(
-        `Your monster is ${newBody}, ${newColor}, and ${newMood}!`
-    );
-};
+function arrayNumberRandomizer(array) {
+    return Math.floor((Math.random() * array.length));
+}
 
-generator(bodyArr,colorArr,moodArr);
\ No newline at end of file
+function arrayRandomizer(array) {
+    return array[arrayNumberRandomizer(array)]
+}
```

Comment:
This way, if your array randomizer logic changes, you change it once, not 3 times